docs/_articles/Power-Production-Distribution-and-Usage-Overview.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: article
+title: Power Production, Distribution, and Usage Overview
+permalink: /articles/Power-Production-Distribution-and-Usage-Overview
+comments: true
+--- 
+### To understand the need for organizations like the REGENCO-OP we must begin with what is wrong with our current methods of producing, distributing and storing energy. The main theme that ties most of these technologies together is the fact that they focus on the current way of producing and distributing power and that’s through large power plants controlled by a power company distributing the power over a large electric grid. For the specific problems we will start with the non-renewable sources first:
+
+* Coal: Coal is mined in both underground mines and in open pits. These mines require huge amounts of energy to extract the ore from the ground. These mines also have very serious environmental impacts that don’t threaten just wildlife but also people directly. There is a lot of coal left so a shortage isn’t foreseen any time soon but by burning all that coal the environmental impact of that would be devastating 
+
+* Natural gas: It’s better than coal but the current rush to extract it uses fracking, a controversial method of pumping water underground at high pressure to break the rock apart and allow the gas to flow to the well.The  USGS has this to say about it:NPR report on USGS statement on fracking causing earthquakes in Oklahoma. There is lots of natural gas available too and because it is overall cleaner than coal will likely be used for quite some time. Again the carbon dioxide and other greenhouse gases would be very detrimental to the environment. 
+
```

Comment:
link needed: NPR report on USGS statement on fracking causing earthquakes in Oklahoma